### DISCUSSION TITLE
nx run without predefined target?

### DISCUSSION THREAD

**Discussion by connormanning** [nrwl/nx#16613] [General]
Hello, perhaps there is an obvious way to do this but I have not found it in the docs nor by fiddling around with the CLI.  I am trying to execute an arbitrary command within a package *without* having a predefined `script` for it in that package's package.json file - is there a way to do this?  For example running `tsc --noEmit` within a single package without defining a script for it.

These attempts do not work:
```
npx nx run mypackage -- tsc --noEmit
npx nx run mypackage:npx -- tsc --noEmit
```

Of course for this particular use-case I could add a script - this is just an example of a more general question.

**Comment by benzsuankularb** [comment]
It's pretty useful.
Did you find any way do achieve this?
